x-pack/test_serverless/api_integration/test_suites/common/alerting/rules.ts

```diff
@@ -45,35 +50,51 @@ export default function ({ getService }: FtrProviderContext) {
     let connectorId: string;
     let ruleId: string;
 
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
```

Comment:
would it be clearer to name this `roleAdmin`?

Reply:
see: https://github.com/elastic/kibana/pull/183381#discussion_r1606716240

Reply:
https://github.com/elastic/kibana/pull/183381/commits/a0c440b9548ddf95a307dabc89bbce536d35de74